keyboards/tada68/keymaps/dlg/config.h

```diff
@@ -0,0 +1,3 @@
+#include "../../config.h"
```

Comment:
This line should be changed to:

```suggestion
#pragma once
```

The build script will include the referenced file automatically without this line, and we only want this file to be processed once.